server/engine/src/main/antlr4/org/eclipse/lsp/db2/parser/Db2SqlParser.g4

```diff
@@ -64,6 +68,8 @@ xml_lobNO_size: BLOB_FILE | CLOB_FILE | DBCLOB_FILE;
 
 xml_as: XML AS;
 
+occurs_clause: OCCURS T=dbs_integer {validateIntegerRange($T.text, 1, 32767);} TIMES?;
```

Comment:
I like above approach of `dbs_host_var_levels_arrays`, where it a single rule for validation. Obviously there is nothing wrong in this as well.